AirCasting/Onboarding/AirBeamOnboarding.swift

```diff
@@ -70,7 +71,7 @@ private extension AirBeamOnboarding {
     private var progressBar: some View {
         ProgressView(value: 0.4)
             .accentColor(.aircastingMint)
-            .padding(.bottom, 20)
+            .padding(.bottom, 30)
```

Comment:
Are you sure we need this padding under the progressView?

Reply:
I think the padding is necessary because on the design we have 30/40px of padding.